templates/skel.html

```diff
@@ -55,7 +55,9 @@
 <body>
     {% set navigation = load_data(path="navigation.toml") %}
     {% include "includes/header.html" %}
-
+    <div id="main-banner">
+        This is the 2024 conference website. For this year, visit <a href="https://2025.matrix.org">2025.matrix.org</a>
```

Comment:
```suggestion
        This is the 2024 conference website. For the current year, visit <a href="https://conference.matrix.org">conference.matrix.org</a>.
```
or we backport my config patch f084900e4b42660965317c3aa0a6040462867ff9 and get
```suggestion
        This is the {{ config.extra.year }} conference website. For the current year, visit <a href="https://conference.matrix.org">conference.matrix.org</a>.
```